Cargo.toml

```diff
@@ -20,7 +20,7 @@ tokio = { version = "1.34", features = [
 ], default-features = false }
 etherparse = { version = "0.13", default-features = false }
 thiserror = { version = "1.0", default-features = false }
-tracing = { version = "0.1.40", default-features = false }
```

Comment:
I'm not sure this is a good choice. I ended up using both in nsproxy. I haven't done enough research on the two although its probably trivial

Reply:
The modifications here are to increase the flexibility of using crates